document/CoreFormat.mdk

```diff
@@ -2425,10 +2425,11 @@ This allows tools to include information about the encountered condition that is
 ### General { #exception-General }
 
 An `exception` object describes a runtime exception encountered in the course of executing an analysis tool.
+This includes signals in operating systems such as POSIX.
```

Comment:
I would say "POSIX conforming operating systems" rather than "operating systems such as POSIX" -- POSIX isn't an operating system, it's an API standard implemented by several operating systems, like BSD and Linux.